### ISSUE TITLE
Only show files that are really moved when using the 'move' command

### ISSUE THREAD

**Issue by fxthomas** [sampsyo/beets#1583]
The `beet move` command is showing *ALL* files in my collection when using the `--pretend` flag, even if the path is the same before and after the `move` operation. Wouldn't it be better to show only the files that are actually moved?

**Comment by sampsyo** [comment]
Sure, sounds great.

**Comment by fxthomas** [comment]
There you go then ;)

**Comment by matrixagent** [comment]
I just came here to report this behavior as well, seeing *all* the files is definitely not what I expected and makes the command a lot scarier for me than it should be. I looked at the PR for this and I'm not quite sure if there is still anything left to do before it can be merged. I'd be happy to help out if someone pointed me in the right direction regarding failing tests e.g.

**Comment by sampsyo** [comment]
The main outstanding issue in #1584 is the need to handle album mode (`beet move -a`). Then we just need some updates to the docs! Thanks being willing to help.

**Comment by matrixagent** [comment]
I checked out the source, not quite figured out how to actually test my changes, though, as it seems running the beet executable from the main directory doesn't work. Haven't worked on any sophisticated Python apps yet. However, as far as I saw from the current master, I think this change should rather be applied inside the show_path_changes method in https://github.com/sampsyo/beets/blob/092472d750fb3d936803be6dacb7ffe09173ade2/beets/ui/__init__.py#L638-L674 – comparing source and destination and hiding output in case they are equal should easily cover items and albums alike.
